server/ai_mediaserver.go

```diff
@@ -505,9 +524,12 @@ func (ls *LivepeerServer) cleanupLive(stream string) {
 	}
 }
 
-const mediaMTXControlPort = "9997"
+const (
+	mediaMTXControlPort = "9997"
+	mediaMTXControlUser = "admin"
+)
 
-func kickInputConnection(sourceID string, sourceType string) error {
+func (ls *LivepeerServer) kickInputConnection(mediaMTXHost, sourceID, sourceType string) error {
```

Comment:
I have been putting live AI specific stuff into [`ai_live_video.go`](https://github.com/livepeer/go-livepeer/blob/master/server/ai_live_video.go) to minimize the sprawl in this file - might be good to do it for this function too, but doesn't have to be done in this PR. I suspect we'll need to call this function in other areas too when we need to force terminate the stream.